src/index.js

```diff
@@ -0,0 +1,232 @@
+'use strict';
+
+const vkey = require('vkey');
+
+const codes = {
+  '<shift>': '',
+  '<backspace>': 'BACKSPACE',
+  '<tab>': 'TAB',
+  '<control>': '',
+  '<escape>': 'ESC',
+  '<alt>': '',
+  '<home>': 'HOME',
+  '<end>': 'END',
+  '<meta>': 'META',
+  '<space>': 'SPACE'
+}
+
+
+function Iggins(app, opts, done){
+
+  let combo = {
+    CTRL_A: {
+      description: 'Select all text in current source code',
+      action: ''
```

Comment:
no need for action here, also shouldn't be a string, also should support multiple registrations